package.json

```diff
@@ -16,7 +16,7 @@
 		"test:unit": "wp-scripts test-unit-js"
 	},
 	"devDependencies": {
-		"@wordpress/scripts": "^19.1.0",
-		"@wordpress/env": "^4.1.3"
+		"@wordpress/env": "^4.1.3",
+		"@wordpress/scripts": "^22.1.0"
```

Comment:
It looks like this prevents errors in the build, though I know this has a vulerability